discuss/advanced/emailtemplate.rst

```diff
@@ -0,0 +1,57 @@
+===============
+Email Templates
+===============
+
+Using email templates let you save time and stay consisted, as you eliminate the need of having to
+rewrite entire emails from scratch for every send. Templates also allow the delivery of
+personalized messages, offering to your recipients a better brand experience.
```

Comment:
I still don't think it's good. 

And there's English mistakes as well 

Here's an idea:

We all know writing good emails is vital to get a high response, but you don't want to rewrite the same structure every time you need to send an email, do you? That's where email templates come in, without the need to rewrite the entire email structure every time you save time to focus on the content. 

Multiple templates will also let you deliver the right message to the right audience, improving their overall experience with the company.